src/Components/Account/Signin.jsx

```diff
@@ -0,0 +1,108 @@
+import React from "react";
+
+const Signin = () => {
```

Comment:
this component should have state change handling as well, as I can see there is no local state being handled for input elements, how would be even record the data being entered by the user and send it to backend ? Please handle it.